README.md

```diff
@@ -1,186 +1,48 @@
-API Application for camptocamp.org v6
-=====================================
+# [camptocamp.org](https://www.camptocamp.org) API
 
-Requirements
-------------
+[![GitHub license](https://img.shields.io/github/license/c2corg/v6_api.svg)](https://github.com/c2corg/v6_api/blob/master/LICENSE) [![Build Status](https://travis-ci.org/c2corg/v6_api.svg?branch=master)](https://travis-ci.com/c2corg/v6_api) [![Total alerts](https://img.shields.io/lgtm/alerts/g/c2corg/v6_api.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/c2corg/v6_api/alerts/) [![Language grade: Python](https://img.shields.io/lgtm/grade/python/g/c2corg/v6_api.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/c2corg/v6_api/context:javascript) [![Codacy Badge](https://api.codacy.com/project/badge/Grade/56217935d9cd43458eb5539ce52a8628)](https://app.codacy.com/app/c2corg/v6_api?utm_source=github.com&utm_medium=referral&utm_content=c2corg/v6_api&utm_campaign=Badge_Grade_Dashboard) [![Known Vulnerabilities](https://snyk.io/test/github/c2corg/v6_api/badge.svg)](https://snyk.io/test/github/c2corg/v6_api)
 
- * Python 3.4
- * PostgreSQL 9.4 / PostGIS 2.1
- * virtualenv (1.7 or higher)
- * ElasticSearch 2.3.2
- * Redis 2.8
- * GEOS (3.4 or higher)
+## Development environment
+On any OS, install [git](https://git-scm.com/) and [docker](https://docs.docker.com/install/). Then :
 
-On Debian/Ubuntu systems the required components may be installed using
+### Install
 
-    sudo apt-get install virtualenv python3-dev postgresql-9.4 postgresql-9.4-postgis-2.1 postgresql-contrib-9.4 libpq-dev libgeos-dev redis-server libffi6 libffi-dev
+```bash
+# Download camptocamp.org source code :
+git clone https://github.com/c2corg/v6_api
+cd v6_api
 
-The Debian package for ElasticSearch can be downloaded from https://www.elastic.co/downloads/past-releases/elasticsearch-2-3-2
+# Build dev env. It's quite long, but you have to do it once.
+docker-compose build
+```
 
-Checkout
---------
+### Run
 
-    git clone https://github.com/c2corg/v6_api.git
+```
+docker-compose up
+```
 
-Build
------
+:heart: [http://localhost:8080](http://localhost:6543) :heart:
 
-    cd v6_api
-    make -f config/{user} install
+Press CTRL+C to terminate it.
 
-To set up the database
-----------------------
+### Check code quality
 
-    scripts/create_user_db.sh
+In another terminal (`docker-compose up` must be running) :
 
-If you want to specify a specific string as user, you can instead use:
+```
+docker-compose exec api make -f config/docker-dev lint
```

Comment:
This indeed works and displays a list of errors if any. But no feedback is provided if no error is found: one might think nothing happened. Displaying an "Code quality check OK" would help.

Reply:
done